### ISSUE TITLE
MPEG2 support roadmap

### ISSUE THREAD

**Issue by B-R-Bender** [phoboslab/jsmpeg#321]
First of all - thanks. You do really crazy things!
Next - I would like to try to implement MPEG2 support in jsmpeg, it's all because of my data is usually MPEG2 encoded, so I'd like to ask: can you, please, give some suggestions or roadmap, I guess, of what should be done, to make jsmpeg be able to handle MPEG2 encoded streams?
Regards, Sergey.

**Comment by winkmichael** [comment]
For all intense and purposes with respect to this project mpeg1 and mpeg2 are basically the same.

**Comment by B-R-Bender** [comment]
yeah, I saw Dominic answer that jsmpeg doesn't respect interlaced video or higher bitrate profiles. However I'd like to have ability to play mpeg2 just because my data is streamed in mpeg2 format, so as I see, I've got two options: implement mpeg2 logic in jsmpeg or convert mpeg2 to mpeg1 which is not actually an option because is a stream data :)

**Comment by whoozle** [comment]
:+1: SD iptv is pain without mpeg2 :(
